### PR TITLE
Create Thoth schema updated command

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Related: https://github.com/thoth-station/thoth-application/issues/2584
https://github.com/AICoE/sefkhet-abwy/issues/184

## This introduces a breaking change

- [x] No

### CODE REVIEW

**Review by harshad16 — APPROVED** [review]
lgtm
thanks :100:

File: aicoe/sesheta/actions/chat.py [hunk 1]
```diff
@@ -76,6 +97,22 @@ async def make_release_issue(request: dict):
     return f"Creating the issue failed. \n Log - {resp_text}"
 
 
+async def make_kebechet_update_issue(repo_name: str) -> str:
```
Comment (codificat) [inline aicoe/sesheta/actions/chat.py:100]: pre-commit is complaining of a missing docstring here

**Review by codificat — CHANGES_REQUESTED** [review]
There is a merge conflict to fix and a missing docstring that pre-commit is complaining about

**Comment by harshad16** [comment]
/remove-lifecycle rotten
/lgtm
/approve

Thank you :100:

**Pull request merged**